src/maingoblint.ml

```diff
@@ -51,15 +51,19 @@ let rec option_spec_list: Arg_complete.speclist Lazy.t = lazy (
       set_string "outfile" "result";
     if get_string "exp.g2html_path" = "" then
       set_string "exp.g2html_path" (Fpath.to_string exe_dir);
-    set_bool "dbg.print_dead_code" true;
+    set_bool "ana.dead-code.lines" true;
+    set_bool "ana.dead-code.branches" true;
+    set_bool "ana.dead-code.functions" true;
```

Comment:
Given that these are the defaults, are they really necessary to force with HTML? If the user explicitly disables these and wants HTML, then they should be disabled. Of course the HTML won't report dead lines then, but that's what the user explicitly wanted, so no surprises there.